website/gatsby-config.js

```diff
@@ -75,6 +75,12 @@ module.exports = {
             componentUrl: resolve(__dirname, '../examples/deck.gl/3d-tiles/app.js'),
             path: 'examples/3d-tiles'
           },
+          {
+            title: 'ArcGIS I3S',
```

Comment:
Why does this say "ArcGIS I3S"? 

Is it using ArcGIS base map? 

We don't call it `Cesium 3D Tiles`...